### ISSUE TITLE
No prop shadows for prop_dynamics

### ISSUE THREAD

**Issue by lvaness** [momentum-mod/game#1272]
**Describe the bug**
This is the long awaited sequel to #1122. Dynamic ~and static~ props don't have shadows currently on Linux.

**To Reproduce**
- Go to the start zone on ahop_coast (on Linux)
- Observe the barrels and crates

**Expected behavior**
The prop should cast a shadow

**Screenshots**
![ahop_coast0000](https://user-images.githubusercontent.com/19615431/111908592-16414c80-8a5a-11eb-994f-5b26db918df6.jpg)

**Desktop/Branch (please complete the following information):**
 - OS: Linux
 - Branch: Latest 0.8.8-rc

**Additional context**
Also verified by Juice on Discord

I tried a few different shader/shadow quality combinations, but nothing changed.

**Comment by tsa96** [comment]
Not seeing prop shadows on ahop_coast on Windows either, they work on 0.8.7. Some shadows from props on ahop_orange work on both 0.8.7 and Chaos, it may be that prop_dynamic entities don't?

**Comment by lvaness** [comment]
Yeah you're probably right, you mean the water tower thing on ahop_orange?

**Comment by Gocnak** [comment]
Imbellis has confirmed these are prop_dynamic_override(s), modified the title to match. Haven't looked into the code behind it yet myself.

**Comment by braem** [comment]
Not seeing prop shadows for any dynamic props (middle is `prop_static`):
![image](https://user-images.githubusercontent.com/9014762/112099901-8686c580-8b61-11eb-82ec-026b3fd87a1e.png)
0.8.7:
![image](https://user-images.githubusercontent.com/9014762/112100002-ad44fc00-8b61-11eb-809d-4d9bcd41a0e4.png)
This is on windows as well.

Map (uses a tf2 prop):
https://cdn.discordapp.com/attachments/781445540386832384/828471588911054858/lighttest.bsp
https://cdn.discordapp.com/attachments/781445540386832384/828471636272742450/lighttest.vmf

**Comment by Gocnak** [comment]
Is this even something we want to fix given there's CSM now?

**Comment by braem** [comment]
> 
> 
> Is this even something we want to fix given there's CSM now?

I'd say yes; CSM is mainly for outdoor lighting to my knowledge and quite a bit of maps are entirely inside. Though i suppose a projected texture might be a better way of doing that.

Also - still occurs on Chaos.
